lib/byron/src/Cardano/Wallet/Byron/Launch.hs

```diff
@@ -163,27 +157,30 @@ withConfig tdir action =
         let nodeSocketFile   = dir </> "node.socket"
         let nodeTopologyFile = dir </> "node.topology"
 
+        -- we need to specify genesis file location every run in tmp
+        configTxt <- T.readFile (source </> "node.config")
+        let configTxtExtended =
+                T.append configTxt ("\nGenesisFile: "<> T.pack nodeGenesisFile)
```

Comment:
wonder if it ("\n") is acceptable for windows env

Reply:
Probably not. Good question. You have two options here IMO:

either, pattern match on `nativeNewLine` from https://hackage.haskell.org/package/base-4.12.0.0/docs/System-IO.html#t:Newline, and put the right mark depending on the result. 

Or, instead of appending a raw line, you could Yaml.decodeFileThrow the node.config, use `withObject` helper from this file, re-encode the modified node.config and write it directly to the temporary config folder (very much like it's node for the genesis file).